care/emr/api/viewsets/diagnostic_report.py

```
@@ -0,0 +1,111 @@
+        if (
+            instance.encounter.facility != instance.facility
+            or instance.service_request.facility != instance.facility
+        ):
+            raise ValidationError(
+                "Diagnostic report facility must be the same as the encounter and service request facility"
+            )
+        return super().perform_create(instance)
+
+    def authorize_create(self, instance):
+        # TODO : AuthZ Pending
+        service_request = get_object_or_404(
```

Ruff (0.8.2)

Local variable `service_request` is assigned to but never used

Remove assignment to unused variable `service_request`

(F841)